rasa/core/channels/facebook.py

```diff
@@ -141,7 +143,21 @@ def send(self, recipient_id: Text, element: Any) -> None:
         # this is a bit hacky, but the client doesn't have a proper API to
         # send messages but instead expects the incoming sender to be present
         # which we don't have as it is stored in the input channel.
-        self.messenger_client.send(element.to_dict(), recipient_id, "RESPONSE")
+        
+        elementlenght = element.to_dict()
+       
+        if 'text' in elementlenght:
+            messagelength = len(elementlenght['text'])
```

Comment:
Please follow the python naming conventions:
- Instance variable names should be all lower case
- Words in an instance variable name should be separated by an underscore

`messagelength` -> `message_length `

Reply:
Hi @tabergma 

I pushed commit 5d6cd39 that should solve this issue